renovate.json

```diff
@@ -1,17 +1,4 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "packageRules": [{
-    "matchSourceUrlPrefixes": ["https://github.com/logto-io/js"],
-    "groupName": "Logto JS SDK monorepo packages"
-  }, {
-    "matchSourceUrlPrefixes": ["https://github.com/logto-io/toolkit"],
-    "groupName": "Logto standard toolkit monorepo packages"
-  }, {
-    "matchSourceUrlPrefixes": ["https://github.com/silverhand-io/configs"],
-    "groupName": "Silverhand configs monorepo packages"
-  }],
-  "extends": [
-    "config:base",
-    ":preserveSemverRanges"
-  ]
+  "extends": ["github>logto-io/.github:renovate-config"]
```

Comment:
Where is this org-level config located?

Reply:
https://github.com/logto-io/.github/blob/master/renovate-config.json